app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<h1>TASKS</h1>
+
+<div class="grid-wrapper">
+  <div class="left-column">
+    <div class="left-buttons">
+      <%= link_to "Create New Task", new_task_path, method: :get %>
+    </div>
+    <div class="left-buttons">
+      <%= link_to "All Tasks", tasks_path %>
+    </div>
+  </div>
+
+  <div class="right-column">
+    <% if @tasks.length > 0 %>
+      <ul>
+        <% @tasks.each do |task| %>
+          <li>
+            <% if task.completed_at.nil? %>
+              <hr><%= link_to task.name, task_path(task.id) %><%=" - " + task.description %>
+              <br />
+              <div class="task-buttons">
+                <%= button_to "🔲 Mark Complete?", complete_task_path(task.id), method: :patch %>
+              </div>
+            <% else %>
+              <hr><s><%= link_to task.name, task_path(task.id) %><%=" - " + task.description %></s>
+              <br />
```

Comment:
I'm being _hilariously_ nit-picky here, but `<s>`, and `<br />` both happen to be HTML tags that are _not_ semantic ;) To get a strikethrough, it might be better to use a class with `text-decoration: strikethrough`, and to get a line-break, it might be better to use a combination of margin and display properties.